src/main/java/hudson/plugins/checkstyle/CheckStylePublisher.java

```diff
@@ -130,19 +146,28 @@ public String getPattern() {
         return pattern;
     }
 
+    /**
+     * Sets the Ant file-set pattern of files to work with.
+     */
+    @DataBoundSetter
+    public void setPattern(final String pattern) {
+        this.pattern = pattern;
+    }
+
     @Override
     public Action getProjectAction(final AbstractProject<?, ?> project) {
         return new CheckStyleProjectAction(project);
     }
 
     @Override
-    public BuildResult perform(final AbstractBuild<?, ?> build, final PluginLogger logger) throws InterruptedException, IOException {
+    public BuildResult perform(final Run<?, ?> build, final FilePath workspace, final PluginLogger logger) throws InterruptedException, IOException {

```

Comment:
@amuniz Cool ... thanks.

Yeah, hard to imagine them being extended/called, but easy to add the method anyway on the 1000000:1 chance :smile: .